packages/components/Button/src/CompoundButton/CompoundButtonTokens.win.ts

```diff
@@ -0,0 +1,166 @@
+import type { Theme } from '@fluentui-react-native/framework';
+import { globalTokens } from '@fluentui-react-native/theme-tokens';
+import { isHighContrast } from '@fluentui-react-native/theming-utils';
+import type { TokenSettings } from '@fluentui-react-native/use-styling';
+
+import type { CompoundButtonTokens } from './CompoundButton.types';
+
+export const defaultCompoundButtonTokens: TokenSettings<CompoundButtonTokens, Theme> = (theme: Theme): CompoundButtonTokens => ({
+  medium: {
+    padding: globalTokens.size120 - globalTokens.stroke.width10,
```

Comment:
>globalTokens.size120 - globalTokens.stroke.width10, [](http://example.com/codeflow?start=13&length=51)

I wonder if this will be needed on windows... we end up having to subtract the width of the border of the button to figure out the padding on win32 because when Views are bound by minHeight/Width instead of just height/width, the border adds to the height of the button so it ends up being bigger than intended (and sometimes causing size changes on focus). This (seemingly?) doesn't happen on web, so I wonder if windows has this problem or not

Reply:
(and we need minHeight for handling text scaling, and minWidth for just who knows how long strings are)